### PR TITLE
Bug 1284385 -  Release history no longer gets data for each revision of a release

### PR DESCRIPTION
**Pull request by njirap**

### CODE REVIEW

File: auslib/admin/views/releases.py [hunk 1]
```diff
@@ -393,10 +393,28 @@ def get(self, release):
...
+        _mapping = {
+            'data_version': 'data_version',
+            'name': 'name',
+            'product': 'product',
+            'read_only': 'read_only',
+            '_different': '_different',
+            '_time_ago': '_time_ago',
+            'change_id': 'change_id',
+            'changed_by': 'changed_by',
+        }
```
Comment (mozbhearsum) [inline auslib/admin/views/releases.py]: What's the purpose of having this mapping when all the keys and values are exactly the same?

**Comment by mozbhearsum** [comment]
It would be good to make sure we don't regress this by adding some test coverage. Perhaps testGetRevisions (https://github.com/mozilla/balrog/blob/master/auslib/test/admin/views/test_releases.py#L945) can be updated to ensure that "data" doesn't exist in the response?

File: auslib/admin/views/releases.py (hunk 1 above)
Comment (mozbhearsum) [inline auslib/admin/views/releases.py]: It also looks like you're missing "timestamp" from here, which makes the releases history items show up with the wrong time in the UI.

**Comment by njirap** [comment]
@mozbhearsum i think we forgot about adding the timestamp..

**Comment by mozbhearsum** [comment]
Adding the timestamp where?

**Comment by njirap** [comment]
You had a comment.."It also looks like you're missing "timestamp" from here, which makes the releases history items show up with the wrong time in the UI."

**Comment by mozbhearsum** [comment]
Ah, right. I just finished merging this manually to master because of the conflicts. I added timestamp ina follow-up (https://github.com/mozilla/balrog/commit/a9ae3813a11db5190c21025fb6f8bc210f995b37).

**Pull request closed without merging**